src/main/java/bit/javaoop/ChiefExecutiveOfficer.java

```diff
@@ -0,0 +1,27 @@
+package bit.javaoop;
+
+import java.util.LinkedList;
+
+public class ChiefExecutiveOfficer extends NamedObject {
```

Comment:
Podoba mi się nazwa klasy, masz szczęście, że to nie Michał Ci piszę review, bo u mnie masz plusa za w pełni rozwinięty skrót :).